configure.py

```diff
@@ -1766,39 +1768,39 @@ def icu_download(path):
     icu_config['variables']['icu_full_canned'] = 1
   # --with-icu-source processing
   # now, check that they didn't pass --with-icu-source=deps/icu
-  elif with_icu_source and os.path.abspath(icu_full_path) == os.path.abspath(with_icu_source):
+  elif with_icu_source and Path(icu_full_path).resolve() == Path(with_icu_source).resolve():
```

Comment:
Ouch! `icu_full_path` can actually be a URL here, and so this change breaks Windows  downloading completely. Not good..